src/android/OSAppFeedback.java

```diff
@@ -242,39 +243,76 @@ public void execute(boolean result) {
     private void registerGestureHandler(){
         final CordovaActivity cordovaActivity = (CordovaActivity) cordova.getActivity();
 
-        webView.getView().setOnTouchListener(new View.OnTouchListener() {
-            @Override
-            public boolean onTouch(View v, MotionEvent event) {
-                int action = MotionEventCompat.getActionMasked(event);
-
-                if(event.getPointerCount() == 2) {
-                    switch (action) {
-                        case MotionEvent.ACTION_POINTER_DOWN:
-                            mSecondFingerTimeDown = System.currentTimeMillis();
-                            mGestureRecognizerTimer = new Timer();
-                            mGestureRecognizerTimer.schedule(new GestureRecognizerTimedTask(cordovaActivity), INTERVAL_TO_SHOW_MENU);
-                            break;
-                        case MotionEvent.ACTION_POINTER_UP:
-                            if ((System.currentTimeMillis() - mSecondFingerTimeDown) <= INTERVAL_TO_SHOW_MENU) {
-                                mGestureRecognizerTimer.cancel();
-                            }
-                            if ((System.currentTimeMillis() - mSecondFingerTimeDown) >= INTERVAL_TO_SHOW_MENU) {
-                                mSecondFingerTimeDown = 0;
+        if(cordovaActivity.getApplicationInfo().targetSdkVersion >= 29) {

```

Comment:
Houston, I think we have a breaking change here.
Although this is protected with the targetSdkVersion for the runtime, I'm not sure if this will build with previous MABS Versions 😞

Reply:
After looking better I believe you are correct. There is a breaking change on the `Constants` import from the Broadcaster. The constants used here (e.g. `Constants.GESTURE_EVENT`) do not exist in older MABS versions (more specifically older Broadcaster versions) and will cause a compile-time error when using this App Feedback commit/version.

A way to avoid this breaking change is to drop the `Constants` import and replace the constants with their string literals, although if the respective values change on the Broadcaster `Constants` class this code can suddenly stop working without visibility.

Regarding the `Event` import from the Broadcaster there is no breaking change because all MABS versions dating back from 3.3 include the Broadcaster with this `Event` interface and the `getData()` method. **Any MABS version using this App Feedback commit/version will compile**. However this is creating a dependency that may break if the `Event` interface is changed...

What are your thoughts on this?

Reply:
The price to pay for not having versioning on this plugin 😞 
Go ahead with that changes.

Reply:
We should aim to have the plugins backwards compatibility with other MABS versions, in this case, it's more justified because we don't have versioning for this plugin. That being said, I agree with @luissilvaos to implement those changes.